addons/sale/models/sale_order.py

```diff
@@ -1420,6 +1350,21 @@ def _create_invoices(self, grouped=False, final=False, date=None):
             )
         return moves
 
+    def _filter_subrecords(self):
```

Comment:
I'd add either a comment section so we understand where there methods come from
`##### Override methods of account.invoiceable.order ####`
Same for SaleOrderLine, PurchaseOrder(Line)

Reply:
It's in an invoicing section, which mostly contains overrides of the mixin, it's enough IMHO.